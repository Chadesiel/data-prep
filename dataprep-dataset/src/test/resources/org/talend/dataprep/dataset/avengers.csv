nickname;secret firstname; secret lastname;date of birth; city
Spiderman;Peter;Parker;;New York City
Superman;Clark;Kent;"02/28/1938";;
Batman;Bruce;Wayne;"05/01/1939";Metropolis
Captain America;Stevens;Rogers;"04/04/1918";NYC
Iron Man;Tony;Stark;"03/05/1963";"N/A"