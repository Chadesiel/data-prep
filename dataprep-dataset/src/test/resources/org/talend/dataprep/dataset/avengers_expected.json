{
  "columns": [
    {
      "id": "nickname",
      "quality": {
        "empty": 0,
        "invalid": 0,
        "valid": 5
      },
      "type": "string"
    },
    {
      "id": "secret firstname",
      "quality": {
        "empty": 0,
        "invalid": 0,
        "valid": 5
      },
      "type": "string"
    },
    {
      "id": "secret lastname",
      "quality": {
        "empty": 0,
        "invalid": 0,
        "valid": 5
      },
      "type": "string"
    },
    {
      "id": "date of birth",
      "quality": {
        "empty": 0,
        "invalid": 0,
        "valid": 5
      },
      "type": "date"
    },
    {
      "id": "city",
      "quality": {
        "empty": 0,
        "invalid": 0,
        "valid": 5
      },
      "type": "string"
    }
  ],
  "records": [
    {
      "nickname": "Spiderman",
      "secret firstname": "Peter",
      "secret lastname": "Parker",
      "date of birth": "05/15/1962",
      "city": "New York City"
    },
    {
      "nickname": "Superman",
      "secret firstname": "Clark",
      "secret lastname": "Kent",
      "date of birth": "02/28/1938",
      "city": "New York City"
    },
    {
      "nickname": "Batman",
      "secret firstname": "Bruce",
      "secret lastname": "Wayne",
      "date of birth": "05/01/1939",
      "city": "NYC"
    },
    {
      "nickname": "Captain America",
      "secret firstname": "Stevens",
      "secret lastname": "Rogers",
      "date of birth": "04/04/1918",
      "city": "New York City"
    },
    {
      "nickname": "Iron Man",
      "secret firstname": "Tony",
      "secret lastname": "Stark",
      "date of birth": "03/05/1963",
      "city": "New York City"
    }
  ]
}